zeebe/clients/java/src/main/java/io/camunda/zeebe/client/impl/ZeebeClientBuilderImpl.java

```diff
@@ -91,6 +101,7 @@ public final class ZeebeClientBuilderImpl implements ZeebeClientBuilder, ZeebeCl
   private String overrideAuthority;
   private int maxMessageSize = 4 * ONE_MB;
   private boolean streamEnabled = false;
+  private boolean grpcAddressUsed = false;
```

Comment:
:wrench: Let's mark it deprecated as we'll remove it when we remove `gatewayAddress` as well.